acceptance/acceptance_test.go

```diff
@@ -229,1905 +99,1417 @@ func testWithoutSpecificBuilderRequirement(
 
-			err = os.Rename(
-				h.CreateTGZ(t, filepath.Join(bpDir, "simple-layers-buildpack"), "./", 0755),
-				filepath.Join(tmpDir, "simple-layers-buildpack.tgz"),
+			buildpackManager.PlaceBuildpacksInDir(
+				tmpDir,
+				components.SimpleLayersParentBuildpack,
+				components.SimpleLayersBuildpack,
 			)
-			h.AssertNil(t, err)
 		})
 
 		it.After(func() {
-			h.AssertNil(t, os.RemoveAll(tmpDir))
+			assert.Nil(os.RemoveAll(tmpDir))
 		})
 
-		packageBuildpackLocally := func(absConfigPath string) string {
-			t.Helper()
-			packageName := "test/package-" + h.RandString(10)
-			output, err := h.RunE(subjectPack("package-buildpack", packageName, "-p", absConfigPath))
-			h.AssertNil(t, err)
-			h.AssertContains(t, output, fmt.Sprintf("Successfully created package '%s'", packageName))
-			return packageName
-		}
-
-		packageBuildpackRemotely := func(absConfigPath string) string {
-			t.Helper()
-			packageName := registryConfig.RepoName("test/package-" + h.RandString(10))
-			output, err := h.RunE(subjectPack("package-buildpack", packageName, "-p", absConfigPath, "--publish"))
-			h.AssertNil(t, err)
-			h.AssertContains(t, output, fmt.Sprintf("Successfully published package '%s'", packageName))
-			return packageName
-		}
-
-		assertImageExistsLocally := func(name string) {
-			t.Helper()
-			_, _, err := dockerCli.ImageInspectWithRaw(context.Background(), name)
-			h.AssertNil(t, err)
-
-		}
-
-		generateAggregatePackageToml := func(buildpackURI, nestedPackageName string) string {
-			t.Helper()
-			packageTomlData := fillTemplate(t,
-				filepath.Join(packFixturesDir, "package_aggregate.toml"),
-				map[string]interface{}{
-					"BuildpackURI": buildpackURI,
-					"PackageName":  nestedPackageName,
-				},
-			)
-			packageTomlFile, err := ioutil.TempFile(tmpDir, "package_aggregate-*.toml")
-			h.AssertNil(t, err)
-			_, err = io.WriteString(packageTomlFile, packageTomlData)
-			h.AssertNil(t, err)
-			h.AssertNil(t, packageTomlFile.Close())
-
-			return packageTomlFile.Name()
-		}
-
 		when("no --format is provided", func() {
 			it("creates the package as image", func() {
 				packageName := "test/package-" + h.RandString(10)
-				output, err := h.RunE(subjectPack("package-buildpack", packageName, "-p", filepath.Join(tmpDir, "package.toml")))
-				h.AssertNil(t, err)
-				h.AssertContains(t, output, fmt.Sprintf("Successfully created package '%s'", packageName))
-				defer h.DockerRmi(dockerCli, packageName)
 
-				assertImageExistsLocally(packageName)
+				output := pack.SuccessfulRunWithOutput(
+					"package-buildpack", packageName,
+					"-p", simplePackageConfigPath,
+				)
+
+				assert.Contains(output, fmt.Sprintf("Successfully created package '%s'", packageName))
+				defer containerManager.RemoveImages(packageName)
+
+				assert.ImageExistsLocally(packageName)
 			})
-		})
 
-		when("--format image", func() {
-			it("creates the package", func() {
+			it("creates the package as a local image", func() {
```

Comment:
This test doesn't seem appropriate for the test, given that it's testing more that package-buildpack can work with a package buildpack as one of the dependencies.

Reply:
I'm not sure I understand the concern? Do you think the test shouldn't exist? Have I misunderstood the test and renamed it incorrectly? I believe this test exists in the current suite, so I wouldn't want to remove it as part of this PR, but happy to do so in another PR if it's not appropriate.

Reply:
I think that I was confused by the naming of the test, because this and the previous one (creates the package as image) both seem to be testing that it can create a package as a local image. Instead, the differentiation between this test and the previous test is that PackageBuildpack can itself consume a packaged buildpack image as one of its dependency buildpacks.